e2e/test_bootstrap_multiple_versions.sh

```
@@ -0,0 +1,74 @@
+#!/bin/bash
+# -*- indent-tabs-mode: nil; tab-width: 2; sh-indentation: 2; -*-
+
+# Test bootstrap with --multiple-versions flag
+# Tests that multiple matching versions are bootstrapped
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "$SCRIPTDIR/common.sh"
+
+# Create constraints file to pin build dependencies (keeps CI fast)
+constraints_file=$(mktemp)
+trap "rm -f $constraints_file" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)